osu.Game/Screens/Play/Player.cs

```diff
@@ -85,6 +90,8 @@ public class Player : ScreenWithBeatmapBackground
 
         private SkipOverlay skipOverlay;
 
+        protected readonly Bindable<OverlayActivation> OverlayActivationMode = new Bindable<OverlayActivation>(OverlayActivation.Disabled);
```

Comment:
seeing that you're binding to `OsuGame`'s bindable bi-directionally, there's no need to expose this for the test - just resolve `OsuGame` in the test itself, get a bound copy from it and inspect that - it should work the same

Reply:
While I agree this should have been the way to do things, this won't work since trying to resolve `OsuGame` in game testing code returns null ( OsuTestBrowser derives from `OsuGameBase`), hence the existence of this protected bindable.

Reply:
fair enough, would have been good to mention it at the start - this however means that this needs xmldoc

also a tiny bit confused at why this is specifying `Disabled` as its initial value but there is also this

https://github.com/ppy/osu/blob/25ebb8619d2cbaa6fc232ed742d33da91b80d703/osu.Game/Screens/Play/Player.cs#L53

Reply:
Bindable initial value should be the same as `InitialOverlayActivationMode` of the screen, I missed that one.